Hunting Queries/Microsoft 365 Defender/Privilege escalation/detect-av-edr-privileged-delete-vulnerability.yaml

```diff
@@ -0,0 +1,25 @@
+id: a9eb9b06-4345-47f2-abe6-29f7200ddf83
+name: Windows Anitivirus and EDR Elevation of Privilege Vulnerability
+description: |
+  The query for malicious file creations via TOCTOU Vulnerability in Leading Endpoint Detection and Response (EDR) and Antivirus (AV) Solutions.
+  - Microsoft Defender (CVE-2022-37971)
+  - Defender for Endpoint (CVE-2022-37971)
+  - SentinelOne EDR
+  - TrendMicro Apex One (CVE-2022-45797)
+  - Avast Antivirus (CVE-2022-4173)
+  - AVG Antivirus (CVE-2022-4173)
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceFileEvents
+tactics:
+- Privilege escalation
```

Comment:
@BlackB0lt, can you please change tactics to "PrivilegeEscalation" and "LateralMovement", thanks.